docs/src/techpapers/netcdf_io.rst

```diff
@@ -0,0 +1,116 @@
+.. _netcdf_io:
+
+.. testsetup:: chunk_control
+
+    import iris
+    from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+
+    from pathlib import Path
+    import shutil
+    import tempfile
+
+    tmp_dir = Path(tempfile.mkdtemp())
+    tmp_filepath = tmp_dir / "tmp.nc"
+
+    cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+    iris.save(cube, tmp_filepath, chunksizes=(120, 37, 49))
+
+
+.. testcleanup:: chunk_control
+
+    shutil.rmtree(tmp_dir)
+
+
+=============================
+NetCDF I/O Handling in Iris
+=============================
+
+This document provides a basic account of how NetCDF files work within Iris.
```

Comment:
This is very broad. `how Iris loads and saves NetCDF files` perhaps? Open to other suggestions.

Reply:
I've not read up on the other headings, so the vaguity was out of ignorance. This line can always be changed by another editor though if it doesn't encompass it all, so this suggestion works for me.